engine/security/trust/trust_sandbox.md

```diff
@@ -74,65 +74,190 @@ the `trustsandbox` container, the Notary server, and the Registry server.
 
 3. Add the following to the new file.
 
-        version: "2"
+        ```yaml
+        version: "2.1"
         services:
-          notaryserver:
-            image: dockersecurity/notary_autobuilds:server-v0.5.1
+          bootstrap:
+            image: golang:1.13-alpine
+            networks:
+              - mdb
+            depends_on:
+              - mysql
             volumes:
-              - notarycerts:/var/lib/notary/fixtures
+              - ./notary/migrations:/go/migrations
+            environment:
+              GO111MODULE: 'on'
+              MIGRATE_VER: v4.6.2
+            command: |-
+              sh -c 'go get -tags 'mysql' github.com/golang-migrate/migrate/v4/cli@$${MIGRATE_VER} &&
+                mv /go/bin/cli /go/bin/migrate &&
+                SERVICE_NAME=notary_signer ./migrations/migrate.sh &&
+                SERVICE_NAME=notary_server ./migrations/migrate.sh'
+
+          server:
+            image: notary:server
             networks:
-              - sandbox
+              mdb:
```

Comment:
With docker-compose version 1.23.1, you need to specify "mdb: {}" if you don't want the interpreter to fail (this is the case for other instances of network).